test/mocha.opts

```diff
@@ -1,2 +1 @@
---bail
```

Comment:
I know this is obnoxious during development, but we might want it during CI? not sure. I've removed it locally myself in the past as well. Maybe we should move it to the script in package.json or whatever?

Reply:
My sentiment is bailing is even less desirable on CI. If 10 tests were failing only in Node 6.x or whatever, I would want to know all of them the first time around rather than discovering one by one.

Reply:
Let's wait until that happens though before changing it. Adding --bail to the package.json.

Reply:
Ah, yeah, I think you're right. Let's just bail on bail (ie; not even include in package.json) for now, unless you think there's good reason to keep it, and can add it back granularly if we find there are times it's useful.